app/experimenter/settings.py

```diff
@@ -19,6 +20,13 @@
 # Build paths inside the project like this: os.path.join(BASE_DIR, ...)
 BASE_DIR = os.path.dirname(os.path.abspath(__file__))
 
+APP_VERSION = ""
+try:
+    version_json = json.load(open(os.path.join(BASE_DIR, "version.json")))
+    APP_VERSION = version_json["commit"]
+except:
+    # No-op, because version.json is usually only generated for real builds
+    pass
```

Comment:
Kind of ugly, tried to work out how to handle the absence of version.json in local dev but when it's available in build.